packages/blockchain-wallet-v4-frontend/src/scenes/TaxCenter/template.tsx

```diff
@@ -106,26 +111,54 @@ const TaxCenter = ({ onGenerateReportClick }: Props) => (
       }
     >
       <Content>
-        <SelectGroup>
-          <SelectLabel size='14px' weight={600} color='black'>
+        <GenerateReport>
+          <SelectGroup>
+            <SelectLabel size='14px' weight={600} color='black'>
+              <FormattedMessage
+                id='scenes.tax.center.card.report.select'
+                defaultMessage='Choose tax year'
+              />
+            </SelectLabel>
+            <StyledSelect
+              value={value}
+              elements={[{ group: '', items: options }]}
+              onBlur={() => {}}
+              onChange={handleChange}
+            />
+          </SelectGroup>
+          <GenerateButton
+            nature='primary'
+            data-e2e='additionalInfoUploadDocument'
+            type='button'
+            onClick={onGenerateReportClick}
```

Comment:
I think the convention is naming triggers with "handle..." for example in this case "handleGenerateReport"